app/Http/Controllers/UsersController.php

```diff
@@ -347,6 +312,48 @@ public function postEdit(UpdateUserRequest $request, $id = null)
     }
 
     /**
+    * Maps Request Information to a User object
+    *
+    * @auther [Daniel Meltzer] [<parallelgrapefruit@gmail.com>]
+    * @since [v3.0]
+    * @param User $user
+    * @param Request $request
+    * @return User
+    */
+    private function extractUserDataFromRequest(User $user, Request $request)
```

Comment:
It's just to centralize the assigning from postCreate and postEdit (because clearly they can get out of sync  :).  It would seem like having all common functionality in one place would make it easier to maintain in the future.  I can copy the information from postEdit to postCreate if that's easier, however.